<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBPlanningHealthResortActivity</class>
 <widget class="QDialog" name="RBPlanningHealthResortActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Планирование санаторной деятельности</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDistrict">
      <attribute name="title">
       <string>Планирование по районам</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblYear">
         <property name="text">
          <string>Год</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="edtYear">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>yyyy</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblRegion">
         <property name="text">
          <string>Регион</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnFillDistricts">
         <property name="text">
          <string>Сгенерировать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="CMainRegionsKLADRComboBox" name="cmbRegion"/>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2" colspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="CInDocTableViewTabMod" name="tblPlan"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOrgStructure">
      <attribute name="title">
       <string>Планирование по отделениям</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3"/>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblCount">
     <property name="text">
      <string>Список пуст</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableViewTabMod</class>
   <extends>QTableView</extends>
   <header>RefBooks/Utils.h</header>
  </customwidget>
  <customwidget>
   <class>CMainRegionsKLADRComboBox</class>
   <extends>QComboBox</extends>
   <header>KLADR/kladrComboxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBPlanningHealthResortActivity</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBPlanningHealthResortActivity</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
